index.js

```diff
@@ -505,103 +502,101 @@
   //. module's `equals` function.
   //.
   //. ```javascript
-  //. > S.equals(0, -0)
+  //. > S.equals (0) (-0)
   //. true
   //.
-  //. > S.equals(NaN, NaN)
+  //. > S.equals (NaN) (NaN)
   //. true
   //.
-  //. > S.equals(S.Just([1, 2, 3]), S.Just([1, 2, 3]))
+  //. > S.equals (S.Just ([1, 2, 3])) (S.Just ([1, 2, 3]))
   //. true
   //.
-  //. > S.equals(S.Just([1, 2, 3]), S.Just([1, 2, 4]))
+  //. > S.equals (S.Just ([1, 2, 3])) (S.Just ([1, 2, 4]))
   //. false
   //. ```
   _.equals = {
     consts: {a: [Z.Setoid]},
     types: [a, a, $.Boolean],
-    impl: Z.equals
+    impl: curry2 (Z.equals)
   };
 
-  //# lt :: Ord a => a -> (a -> Boolean)
+  //# lt :: Ord a => a -> a -> Boolean
   //.
   //. Returns `true` [iff][] the *second* argument is less than the first
-  //. according to [`Z.lt`][]. The arguments must be provided one at a time.
+  //. according to [`Z.lt`][].
```

Comment:
This is one of the important changes listed in <https://github.com/sanctuary-js/sanctuary/issues/438#issuecomment-371809619>. Now that all functions are “simply” curried there's no reason for this function's type to be `[a, $.Predicate (a)]` rather than `[a, a, $.Boolean]`. :)